ext/moderation.py

```diff
@@ -0,0 +1,224 @@
+import disnake
+from disnake.ext import commands
+
+from utils.bot import Cog
+from utils.converter import TimeConverter as convert
+
+
+class moderation(Cog):
+    @commands.slash_command(name="ban")
+    @commands.has_permissions(ban_members=True)
+    async def ban(
+        self, inter: disnake.ApplicationCommandInteraction, member: disnake.Member, reason: str = "No reason"
+    ):
+        """Bans a member
+
+        Parameters
+        -----------
+        member: The member to ban
+        reason: The reason for banning
+        """
+        if inter.user.top_role > member.top_role:
```

Comment:
Um yeah sorry to inform you but it should be like this
```py
if inter.user.top_role <= member.top_role:
    raise HierarchyError()  # define it in utils.errors
```
Also better to bring that out to a function

Reply:
cant i just use else statement and raise error?